torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -569,14 +573,6 @@ def rotate_bounding_box(
         expand=expand,
     )
 
-    if expand:
-        # TODO: Move this computation inside of `_affine_bounding_box_xyxy` to avoid computing the rotation and points
-        #  matrix twice
-        height, width = image_size
-        rotation_matrix = _get_inverse_affine_matrix([0.0, 0.0], angle, [0.0, 0.0], 1.0, [0.0, 0.0])
```

Comment:
Here we passed center=[0, 0] instead of the value of `center`. Was that a bug? None of the tests captured a difference locally. This might indicate a gap on our testing strategy. cc @pmeier

Reply:
Also the value of `inverted` was True, while on the actual kernel we pass explicitly `False`.

Reply:
Center and expand are orthogonal, if expand is True, center should be default = in the image center.

Reply:
Should we explicitly enforce and align this in the `_affine_bounding_box_xyxy` method?

Reply:
OK sounds like there is no issue with `center`. How about the value of the `inverted`?